source/Applications/Advanced/MaskPointCloud/MaskPointCloud.cpp

```diff
@@ -0,0 +1,224 @@
+/*
+This example shows how to mask a point cloud from ZDF file and store
+the resulting point cloud in PCL format. The ZDF file for this sample can be found under the
+main instructions for Zivid samples.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <opencv2/opencv.hpp>
+
+#include <pcl/visualization/cloud_viewer.h>
+
+#include <iostream>
+
+namespace
+{
+    void visualizePointCloud(const pcl::PointCloud<pcl::PointXYZRGB> &pointCloud)
+    {
+        // Simple Cloud Visualization
```

Comment:
Don't think this line is needed?

Reply:
In the bottom of this page it says that line: https://zividlabs.atlassian.net/wiki/spaces/CS/pages/1115488261/Samples+standardization+-+printouts+comments 
Shall we edit it away?

Reply:
That is an example. I think the first thing should be to decide whether you print out or comment in functions. Then, apply is decided. If the reason to keep this is consistency I can agree with that.